native/c/makefile

```diff
@@ -88,178 +97,220 @@ MTL_FLAGS+=-g1
 MTL_FLAGS64+=-g1
 .END
 
+all: libzut.so libzut.a libzds.so libzds.a libzusf.so libzusf.a libzcn.so libzcn.a libzjb.so libzjb.a zowex zoweax extattr test
+xlclang-extenders: $(OUT_DIR_CXXLANG)/zut.o $(OUT_DIR_CXXLANG)/zcn.o
 
-all: native zowex zoweax extattr test
-native: libzut.so libzut.a libzds.so libzds.a libzusf.so libzusf.a libzcn.so libzcn.a libzjb.so libzjb.a
```

Comment:
I liked the cleaner `native` look 😋

Reply:
me too - that was a mistake lol we can bring it back if/when there is a need